tests/chainer_tests/functions_tests/loss_tests/test_huber_loss.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class TestHuberLoss(unittest.TestCase):
+
+    def setUp(self):
+        self.shape = (4, 10)
+        self.x = (numpy.random.random(self.shape) - 0.5) * 20
+        self.x = self.x.astype(numpy.float32)
+        self.t = numpy.random.random(self.shape).astype(numpy.float32)
+
+    def check_forward(self, x_data, t_data):
+        x = chainer.Variable(x_data)
+        t = chainer.Variable(t_data)
+        loss = functions.huber_loss(x, t, delta=1)
+        self.assertEqual(loss.data.dtype, numpy.float32)
+        loss_value = cuda.to_cpu(loss.data)
+
+        diff_data = cuda.to_cpu(x_data) - cuda.to_cpu(t_data)
+        expected_result = numpy.zeros(self.shape)
+        mask = numpy.abs(diff_data) < 1
+        expected_result[mask] = 0.5 * diff_data[mask]**2
+        expected_result[~mask] = numpy.abs(diff_data[~mask]) - 0.5
+        loss_expect = numpy.sum(expected_result, axis=1)
+        numpy.testing.assert_allclose(loss_value, loss_expect)
```

Comment:
Would you relax the threshold of testing? It seems be failed sometimes. Just using gradient_check.assert_allclose is also good, since it uses more loose thresholds by default.